Sl/w 16972747 (#1257)
salesforcecli/plugin-org: messages/delete_scratch.md

# error.missingUsername

Unable to determine the username of the org to delete. Specify the username with the --target-org | -o flag.

# error.unknownScratch

> The org with username: %s is not known by the CLI to be a scratch

Reviewer: ```suggestion
The org with username %s isn't a scratch org. 
```

Reviewer: @soridalac , okay?

Reviewer: Alternatively, how about:

Unable to find a scratch org with username %s.

Author: okay thats work, I will change it.
